catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/validation/impl/validator/MatchAnyValidator.java

```diff
@@ -37,42 +39,62 @@ public MatchAnyValidator(List<AttributeValidator> validators) {
       return Optional.empty();
     }
 
-    List<Optional<AttributeValidationReport>> validationReportList =
-        validators
-            .stream()
-            .map(validator -> validator.validate(attribute))
-            .collect(Collectors.toList());
+    List<Serializable> serializables = attribute.getValues();
```

Comment:
nit: serializables? It is attributeValues or values or something...

Reply:
done